apollo-audit/apollo-audit-impl/src/main/java/com/ctrip/framework/apollo/audit/aop/ApolloAuditSpanAspect.java

```diff
@@ -48,49 +48,45 @@ public void setAuditSpan(ApolloAuditLog auditLog) {
   @Around(value = "setAuditSpan(auditLog)")
   public Object around(ProceedingJoinPoint pjp, ApolloAuditLog auditLog) throws Throwable {
     String opName = auditLog.name();
-    if (opName.equals("") && RequestContextHolder.getRequestAttributes() != null) {
-      ServletRequestAttributes servletRequestAttributes = (ServletRequestAttributes) RequestContextHolder.getRequestAttributes();
-      opName = servletRequestAttributes.getRequest().getRequestURI();
-    }
-
     try (AutoCloseable scope = api.appendAuditLog(auditLog.type(), opName,
         auditLog.description())) {
-      Object[] args = pjp.getArgs();
-      Method method = findMethod(pjp.getTarget().getClass(), pjp.getSignature().getName());
-      for (int i = 0; i < args.length; i++) {
-        Object arg = args[i];
-        Annotation[] annotations = method.getParameterAnnotations()[i];
-        if (Arrays.stream(annotations).anyMatch(anno -> anno instanceof ApolloAuditLogDataInfluence)) {
-          String entityName = null;
-          String fieldName = null;
-          for(int j = 0; j < annotations.length; j++) {
-            if(annotations[j] instanceof ApolloAuditLogDataInfluenceTable) {
-              entityName = ((ApolloAuditLogDataInfluenceTable) annotations[j]).tableName();
-            }
-            if(annotations[j] instanceof ApolloAuditLogDataInfluenceTableField) {
-              fieldName = ((ApolloAuditLogDataInfluenceTableField) annotations[j]).fieldName();
-            }
-          }
-          if (entityName != null && fieldName != null) {
-            // if arg is a collection
-            if (arg instanceof Collection) {
-              for (Object o : ((Collection<?>) arg).toArray()) {
-                String matchedValue = String.valueOf(o);
-                api.appendDataInfluence(entityName, "AnyMatched", fieldName, matchedValue);
-              }
-            }
-            else {
-              String matchedValue = String.valueOf(arg);
-              api.appendDataInfluence(entityName, "AnyMatched", fieldName, matchedValue);
-            }
-          }
+      auditDataInfluenceArg(pjp);
+      return pjp.proceed();
```

Comment:
Only record the audit log after method executed success,

so auditDataInfluenceArg should after pjp.proceed()?